### ISSUE TITLE
Bug: Changing the font size when inputting does not take effect

### ISSUE THREAD

**Issue by zhuba-Ahhh** [facebook/lexical#6000]
Lexical version: latest

## Steps To Reproduce

1. open https://playground.lexical.dev/
2. Change font size when inputting

https://github.com/facebook/lexical/assets/84793349/5d13a4a3-f4f7-44fa-833a-3416708fa2db

Link to code example:

## The current behavior
Changing the font size when inputting does not take effect

## The expected behavior
Input can change font size

**Comment by CARPEDIEM05** [comment]
Hi @StyleT , I want to try this. This is the first time I am contributing to open-source. How can I assign this to myself?

**Comment by zhuba-Ahhh** [comment]
> Hi @zhuba-Ahhh @StyleT, I somewhat got the solution. It is taking effect but we have to press enter as it is been stated in onKeyDown that you have to press enter to change the state of the font size.

Yes, I have found that pressing Enter is effective, but simply typing to change the value in the input box is not effective. You can try

**Comment by CARPEDIEM05** [comment]
As like we are handling the onChange and onKeyDown event, we have to handle, onBlur event. My suggestion is to remove onKeyDown handler and just use the onBlur event to handle the change of the font.

**Comment by zhuba-Ahhh** [comment]
> Hi @zhuba-Ahhh As like we are handling the onChange and onKeyDown event, we have to handle, onBlur event. My suggestion is to remove onKeyDown handler and just use the onBlur event to handle the change of the font.

Yes, you can try it like this?

<img width="1093" alt="image" src="https://github.com/facebook/lexical/assets/84793349/c263c949-62ca-4376-958f-48a3b8519d47">

**Comment by CARPEDIEM05** [comment]
You might be right, who will be the right person to comment on this? You or someone else? I am new to this so I don't know.

**Comment by CARPEDIEM05** [comment]
According to me
1. Either to remove the onKeyDown event and add the onBlur event
2. We can include both the event


What do you think?

**Comment by StyleT** [comment]
> Hi @StyleT , I want to try this. This is the first time I am contributing to open-source. How can I assign this to myself?

Just comment here that you'll work on it and submit a PR ;) Even if 2+ people submit concurrent PRs (never happens, almost) - it's ok.

**Comment by CARPEDIEM05** [comment]
Hey @StyleT , I am working on it, i have submitted my code for review. So hope so that it will be assign to me.

**Comment by kirandash** [comment]
I cannot reproduce this issue. This appears to be working fine and can be closed.

https://github.com/user-attachments/assets/51104414-41b2-4a28-9520-57e486b86405
